azure_arc_data_jumpstart/gke/mssql_mi/terraform/scripts/ClientTools.ps1

```diff
@@ -0,0 +1,207 @@
+Start-Transcript -Path C:\tmp\ClientTools.log
+
+# Installing Chocolatey packages
+$chocolateyAppList = "azure-cli,az.powershell,kubernetes-cli,vcredist140"
+
+if ([string]::IsNullOrWhiteSpace($chocolateyAppList) -eq $false)
+{
+    try{
+        choco config get cacheLocation
+    }catch{
+        Write-Output "Chocolatey not detected, trying to install now"
+        iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+    }
+}
+
+if ([string]::IsNullOrWhiteSpace($chocolateyAppList) -eq $false){   
+    Write-Host "Chocolatey Apps Specified"  
+    
+    $appsToInstall = $chocolateyAppList -split "," | foreach { "$($_.Trim())" }
+
+    foreach ($app in $appsToInstall)
+    {
+        Write-Host "Installing $app"
+        & choco install $app /y | Write-Output
+    }
+}
+
+# Downloading Azure Data Studio and azdata CLI
+Write-Host "Downloading Azure Data Studio and azdata CLI"
+Write-Host "`n"
+Invoke-WebRequest "https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-archive/stable" -OutFile "C:\tmp\azuredatastudio.zip" | Out-Null
+Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_arc/gke_sqlmi/azure_arc_data_jumpstart/gke/mssql_mi/terraform/settings.json" -OutFile "C:\tmp\settings.json"
+Invoke-WebRequest "https://aka.ms/azdata-msi" -OutFile "C:\tmp\AZDataCLI.msi" | Out-Null
+Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_arc/gke_sqlmi/azure_arc_data_jumpstart/gke/mssql_mi/terraform/scripts/MSSQL_MI_Cleanup.ps1" -OutFile "C:\tmp\MSSQL_MI_Cleanup.ps1"
+Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_arc/gke_sqlmi/azure_arc_data_jumpstart/gke/mssql_mi/terraform/scripts/MSSQL_MI_Deploy.ps1" -OutFile "C:\tmp\MSSQL_MI_Deploy.ps1"
```

Comment:
```suggestion
Invoke-WebRequest "https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-archive/stable" -OutFile "C:\tmp\azuredatastudio.zip" | Out-Null
Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/gke/mssql_mi/terraform/settings.json" -OutFile "C:\tmp\settings.json"
Invoke-WebRequest "https://aka.ms/azdata-msi" -OutFile "C:\tmp\AZDataCLI.msi" | Out-Null
Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_arc/gke_sqlmi/azure_arc_data_jumpstart/gke/main/terraform/scripts/MSSQL_MI_Cleanup.ps1" -OutFile "C:\tmp\MSSQL_MI_Cleanup.ps1"
Invoke-WebRequest "https://raw.githubusercontent.com/microsoft/azure_arc/gke_sqlmi/azure_arc_data_jumpstart/gke/main/terraform/scripts/MSSQL_MI_Deploy.ps1" -OutFile "C:\tmp\MSSQL_MI_Deploy.ps1"
```

Reply:
This isnt 100% correct and would result in broken links. I will change this after merge.